src/components/Questions/QuestionModal.vue

```diff
@@ -1,39 +1,53 @@
 <template>
-  <div
-    class="flex flex-col bg-white w-full h-full justify-between overflow-hidden"
-  >
-    <Body
-      class="mt-10"
-      :text="currentQuestion.text"
-      :options="currentQuestion.options"
-      :correctAnswer="questionCorrectAnswer"
-      :questionType="questionType"
-      :isGradedQuestion="isGradedQuestion"
-      :maxCharLimit="currentQuestion.max_char_limit"
-      :isPortrait="isPortrait"
-      :imageData="currentQuestion?.image"
-      :draftAnswer="draftResponses[currentQuestionIndex]"
-      :submittedAnswer="currentQuestionResponseAnswer"
-      :isAnswerSubmitted="isAnswerSubmitted"
-      @option-selected="questionOptionSelected"
-      @answer-entered="subjectiveAnswerUpdated"
-      data-test="body"
-    ></Body>
-    <Footer
-      :isAnswerSubmitted="isAnswerSubmitted"
-      :isPreviousButtonShown="currentQuestionIndex > 0"
-      :isSubmitEnabled="isAttemptValid"
-      @submit="submitQuestion"
-      @continue="showNextQuestion"
-      @previous="showPreviousQuestion"
-      data-test="footer"
-    ></Footer>
+  <div class="h-full flex flex-col">
+    <Header
+      v-if="isQuizAssessment"
+      @end-test="endTest"
+      :hasQuizEnded="hasQuizEnded"
+    ></Header>
+    <div
+      class="flex flex-col grow bg-white w-full justify-between overflow-hidden"
+    >
+      <Body
+        class="mt-10"
+        :text="currentQuestion.text"
+        :options="currentQuestion.options"
+        :correctAnswer="questionCorrectAnswer"
```

Comment:
rename to `isCorrectAnswer`?

Reply:
this prop does not indicate if the answer is correct or not - it passes the raw correct answer.